src/docker-osx-dev

```diff
@@ -787,6 +801,8 @@ function instructions {
   echo -e
   echo -e "Commands:"
   echo -e "  $SYNC_COMMAND\t\tStart file syncing. This is the default if no COMMAND is specified."
+  echo -e "  $WATCH_ONLY_COMMAND\tWatch the file system for changes, which syncing first."
+  echo -e "  $SYNC_ONLY_COMMAND\tSync the file system without watching afterwards."
```

Comment:
Perhaps slightly clearer: "Sync the file system and exit, without watching afterwords"